var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -126,7 +179,7 @@ class Trilinos(CMakePackage):
     depends_on('superlu-dist', when='@12.6.2:+superlu-dist')
     depends_on('superlu-dist@develop', when='@develop+superlu-dist')
     depends_on('superlu-dist@xsdk-0.2.0', when='@xsdk-0.2.0+superlu-dist')
-    depends_on('superlu+fpic@4.3', when='+superlu')
+    depends_on('superlu@4.3', when='+superlu')
```

Comment:
Can this package be built with `superlu~fpic`? That's what this change suggests.